images/20-zfs/build.sh

```diff
@@ -121,7 +121,23 @@ done
 for i in $(ls arch/sbin); do
    echo "system-docker cp wonka.sh \${1}:/sbin/$i" >> /dist/arch/setup_wonka.sh
 done
-system-docker build --network=host -t zfs-tools arch/
+
+if [ "XX$HTTP_PROXY" != "XX" ]
+then
+	BUILD_ARGS="--build-arg HTTP_PROXY --build-arg http_proxy="$HTTP_PROXY
```

Comment:
Can you indent with 4 spaces? We want to be consistent.

Also, we are more like to use:
```
if [ "XX$HTTP_PROXY" != "XX" ]; then

fi
```